dev/com.ibm.ws.anno/resources/com/ibm/ws/anno/resources/internal/AnnoMessages.nlsprops

```diff
@@ -445,15 +445,15 @@ ANNO_CLASSSOURCE_ENTRY_SCAN_EXCEPTION.useraction=See the exception and the assoc
 # # {4} The error message from the exception
 ANNO_CLASSSOURCE_ENTRY_CLOSE_EXCEPTION=CWWKC0068W: [ {0} ]: The close of entry [{1}] for class [{2}] in [{3}] failed. Error message: {4}
 ANNO_CLASSSOURCE_ENTRY_CLOSE_EXCEPTION.explanation=The close of the entry caused an exception.
-ANNO_CLASSSOURCE_ENTRY_CLOSE_EXCEPTION.useraction=See the exception and the associated FFDC message for more information.
+ANNO_CLASSSOURCE_ENTRY_CLOSE_EXCEPTION.useraction=For more information on the topic, see the exception and the associated FFDC message.
```

Comment:
Change
'For more information on the topic, see the exception and the associated FFDC message.'
to
'For more information, see the exception and the associated FFDC message.'

Reply:
Updated.